### PR TITLE
Outbox cleanup

### PR DESCRIPTION
**Pull request by swistak35**

### CODE REVIEW

File: contrib/ruby_event_store-outbox/Gemfile [hunk 1]
```diff
@@ -13,11 +13,7 @@ gem 'influxdb', '~> 0.8.0', require: false
 gem 'childprocess'
 gem 'rails', ENV['RAILS_VERSION']
```
Comment (pawelpacana) [inline contrib/ruby_event_store-outbox/Gemfile:14]: RAILS_VERSION is more, multiple Gemfiles are preferred (each having a lockfile)

Comment (swistak35) [reply contrib/ruby_event_store-outbox/Gemfile:14]: I will start with one version working and expand later

Comment (pawelpacana) [reply contrib/ruby_event_store-outbox/Gemfile]: s/ENV['RAILS_VERSION']/6.0.3.4/ then

File: .github/workflows/ruby_event_store-outbox.yml [hunk 2]
```diff
@@ -26,3 +26,30 @@ jobs:
...
         working-directory: ${{ env.GEM_LOCATION }}
+
+  mutate:
+    runs-on: ubuntu-20.04
+    env:
+      GEM_NAME: ruby_event_store-outbox
+      BUNDLE_GEMFILE: ${{ matrix.gemfile }}
+      DATABASE_URL: ${{ matrix.database }}
+      GEM_LOCATION: contrib/ruby_event_store-outbox
+      MUTANT_JOBS: 1
```
Comment (pawelpacana) [inline .github/workflows/ruby_event_store-outbox.yml:42]: should not be needed when `sqlite3::memory:`

**Pull request merged**